lib/Service/Social/TumblrProvider.php

```diff
@@ -35,6 +35,7 @@ public function __construct() {
 	 * @return string
 	 */
 	public function cleanupId(string $candidate):?string {
+		$candidate = preg_replace('/^' . preg_quote('x-apple:', '/') . '/', '', $candidate);
```

Comment:
Maybe this should go into the level above so $candidate is already sanitized?

Reply:
https://github.com/nextcloud/contacts/blob/709615173f3b38dbea2331a530c1edd5bb68f880/lib/Service/Social/CompositeSocialProvider.php#L82

Reply:
Sure, it can be there, too. But apple only attaches this to custom fields. For certain networks (for example Facebook and Twitter) it adds the domain of the service. So because it is not valid for all networks, I only added it where necessary. I'll let you decide where it's best.

Reply:
Well, I guess it's easier to have it once and for all rather than on every provider?
Maybe I'm missing something :thinking: :wink:

Reply:
just to be clear: we do not need it for every provider. apple only adds this "x-apple" thingy to the networks it does not know. All pre-defined networks don't have that tag. So I thought its more a temporary thing and it can be removed time after time from the known networks.